### ISSUE TITLE
Is this still viable to work with?

### ISSUE THREAD

**Issue by ottomansky** [dolanmiu/steam-analyst#1]
Is this ready to be built upon or are things that much changed that its basically useles?

**Comment by dolanmiu** [comment]
No, this project hasn't been updated in a while. Probably best to use this project as inspiration, but not directly to be used!
